parcels/field.py

```diff
@@ -171,6 +173,7 @@ def __init__(self, name, data, lon=None, lat=None, depth=None, time=None, grid=N
         self.loaded_time_indices = []
         self.creation_log = kwargs.pop('creation_log', '')
         self.field_chunksize = kwargs.pop('field_chunksize', None)
+        self.netcdf_name_maps = kwargs.pop('name_maps', None)
```

Comment:
Again, add the word 'chunking' to this variable name? E.g. `self.chunking_dims_name_map`?

Reply:
agreed.